### ISSUE TITLE
NTR: 5-hydroxyhexanoic acid high in urine (Lab: chemistry: organic acids)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8597]
New term request 5-hydroxyhexanoic acid high in urine (segal_030513143849):

**Comment by pnrobinson** [comment]
@MickeySegal

Is this (5R)-5-Hydroxyhexanoic acid. If so, what is the clinical relevance?

https://hmdb.ca/metabolites/HMDB0000409
	(5R)-5-Hydroxyhexanoic acid, also known as (5R)-5-hydroxycaproate or HMBA, belongs to the class of organic compounds known as medium-chain fatty acids. These are fatty acids with an aliphatic tail that contains between 4 and 12 carbon atoms. Based on a literature review very few articles have been published on (5R)-5-Hydroxyhexanoic acid.

**Comment by MickeySegal** [comment]
https://www.ncbi.nlm.nih.gov/books/NBK558236/
"Urine organic acid analysis shows elevations of multiple organic acids, including:...
5-hydroxyhexanoic acid"

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000778
- label: Elevated urinary 5-hydroxyhexanoic acid level
- parents: [Elevated urinary carboxylic acid (HP:0040156)](https://hpo.jax.org/app/browse/term/HP:0040156)
- pmid: PMID:32550677
- def: The amount of 5-hydroxyhexanoic acid in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8597
- synonyms:

**Comment by pnrobinson** [comment]
@MickeySegal 
sorry the identifier is

HP:0025734

I had to correct a mistake.
